### ISSUE TITLE
Dismap support

### ISSUE THREAD

**Issue by ivarocas** [ivre/ivre#1597]
Hi Pierre,

Is still dismap supported? We attempted to  import the output of dismap in json format, but this error appears:

```
WARNING:ivre:Exception (file 'output.json')
Traceback (most recent call last):
File "/usr/local/lib/python3.9/dist-packages/ivre/db/init.py", line 2148, in store_scan
store_scan_function = {
KeyError: b'{'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
File "/usr/local/lib/python3.9/dist-packages/ivre/tools/[scan2db.py](http://scan2db.py/)", line 146, in main
if [database.store](http://database.store/)_scan(
File "/usr/local/lib/python3.9/dist-packages/ivre/db/init.py", line 2188, in store_scan
raise ValueError( # pylint: disable=raise-missing-from
ValueError: Unknown file type output.json
INFO:ivre:0 results imported.
```
We tried all of versions of dismap available.

Best Regards

**Comment by p-l-** [comment]
Hi there!
Yes, Dismap is supposed to be supported. Could you share a result you are trying to insert?
Thanks!

**Comment by ivarocas** [comment]
Hi Pierre,

Sorry for the delay of my answer. Here the file that we tried to import: [output.json](https://github.com/ivre/ivre/files/13885821/output.json) 

We tried some others outputs but the result was the same.

Many Thanks!

**Comment by p-l-** [comment]
Hi @ivarocas,

It seems that Dismap has switched from JSON to NDJSON format. This should be fixed in #1604. Can you try it and report if that fixes your issue? At least it works with the sample you provided.
